dll/cpl/desk/lang/tr-TR.rc

```diff
@@ -1,4 +1,11 @@
-/* TRANSLATOR: 2013-2020 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net> */
+/*
+ * PROJECT:     ReactOS Display Control Panel
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Turkish resource file
+ * TRANSLATORS: Copyright 2013-2020 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net>
+ *              Copyright 2021-2023 Süleyman Poyraz <zaryob.dev@gmail.com>
+ *              Copyright 2024 Ethem Çılgın (fiatcoupe16vturbo) <ethemrty@gmail.com>
```

Comment:
```suggestion
 *              Copyright 2024 Ethem Çılgın <ethemrty@gmail.com>
```
I think we can omit nicknames here for new contributors, since it gets stored in Git anyway. This was actual in the SVN times when some patches were sent via Jira.